README.md

```diff
@@ -245,4 +245,4 @@ Distributed under the Apache2 License. Read the `LICENSE` file for more informat
 
 [product-screenshot]: .github/demo.svg
 
-[feature-py-def]: .github/feature-py-def.png
\ No newline at end of file
+[feature-py-def]: https://i.ibb.co/Nm3Wbdm/feature-py-def.png
```

Comment:
Why ImgBB is better than managing the image in the git? It's much simpler and transparent(for PRs) to replace it via a commit